### PR TITLE
Added missing dependecies

### PR DESCRIPTION
**Pull request by FlightBlaze**
- added missing dependecies & dependency install guide in README.md

This will help compiling sources for new contributors

### CODE REVIEW

**Comment by opnay** [comment]
These some dependencies are included on `elementary-sdk` package
```
sudo apt install elementary-sdk
```
see more (https://elementary.io/docs/code/getting-started#developer-sdk)

File: README.md [hunk 1]
```diff
@@ -43,9 +43,16 @@ You can install Akira by compiling it from the source, here's the list of depend
...
  - `gtksourceview-3.0`
+ - `libjson-glib-1.0`
  - `goocanvas-2.0`
+ - `libarchive`
  - `cairo`
  - `meson`
+ 
+ In Elementary OS 0.3 Freya and later, resolve dependencies by typing:
+ ```
+ sudo apt-get install -y valac meson build-essential libgtk-3-dev libgranite-dev libxml++2.6-dev libgoocanvas-2.0-dev libarchive-dev glib-2.0 libgtksourceview-3.0-dev libjson-glib-dev
```
Comment (bilelmoussaoui) [inline README.md]: Can you please remove this part? The readme shouldn't ask anyone to execute anything as a sudoer.

Comment (FlightBlaze) [reply README.md]: Ok

**Comment by Alecaddd** [comment]
Thanks for your contribution.

**Pull request merged**